### PR TITLE
[AWS] Add EMR logs dashboard

### PR DESCRIPTION
**Pull request by ghost**
## What does this PR do?

Adds EMR logs dashboard. Also added `data_stream.dataset` filter.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

## Screenshots
emr_logs

### CODE REVIEW

**Review by nickofthyme — APPROVED** [review]
JSON configuration LGTM 👍🏼

File: packages/aws/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.50.2"
+  changes:
+    - description: Add EMR logs dashboard.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/6895
```
Comment (kaiyan-sheng) [inline packages/aws/changelog.yml:6]:
```suggestion
      link: https://github.com/elastic/integrations/pull/7075
```
and should this be 1.50.1 instead?

**Review by kaiyan-sheng — APPROVED** [review]
LGTM, thanks for adding the screenshot.

**Pull request merged**